docs/tutorial/security.md

```diff
@@ -723,6 +723,41 @@ Migrate your app one major version at a time, while referring to Electron's
 [Breaking Changes][breaking-changes] document to see if any code needs to
 be updated.
 
+### 17. Validate the `sender` of all IPC messages
+
+You should always validate incoming IPC messages `sender` property to ensure you
+aren't performing actions or sending information to untrusted renderers.
+
+#### Why?
+
+All Web Frames can in theory send IPC messages to the browser process, this includes
+iframes and child windows in some scenarios.  If you have an IPC message that returns
+user data to the sender via `event.reply` or performs privileged actions that the renderer
+can't natively you should ensure you aren't listening to third party web frames.
```

Comment:
```suggestion
All Web Frames can in theory send IPC messages to the main process, including
iframes and child windows in some scenarios.  If you have an IPC message that returns
user data to the sender via `event.reply` or performs privileged actions that the renderer
can't natively, you should ensure you aren't listening to third party web frames.
```

Reply:
Also: we don't really use the term "web frame" a lot in our docs. Is this equivalent to "web contents" or is there a distinction here that's important?

Reply:
The distinction is important, a web contents hosts one-to-many web frames